README.md

```diff
@@ -37,42 +37,81 @@ DeepRank2 extensive documentation can be found [here](https://deeprank2.rtfd.io/
   - [Overview](#overview)
   - [Table of contents](#table-of-contents)
   - [Installation](#installation)
-    - [Dependencies](#dependencies)
-    - [Deeprank2 Package](#deeprank2-package)
-    - [Test installation](#test-installation)
+    - [Dockerfile](#dockerfile)
+    - [Non-pythonic dependencies](#non-pythonic-dependencies)
+    - [Pythonic dependencies](#pythonic-dependencies)
+      - [Test installation](#test-installation)
     - [Contributing](#contributing)
-    - [Data generation](#data-generation)
-    - [Datasets](#datasets)
-      - [GraphDataset](#graphdataset)
-      - [GridDataset](#griddataset)
-    - [Training](#training)
+  - [Data generation](#data-generation)
+  - [Datasets](#datasets)
+    - [GraphDataset](#graphdataset)
+    - [GridDataset](#griddataset)
+  - [Training](#training)
   - [Computational performances](#computational-performances)
   - [Package development](#package-development)
 
 ## Installation
 
-The package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
+Note that the package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 
```

Comment:
I think it would be nice to give a short summary of the different options here.

```suggestion
Note that the package officially supports ubuntu-latest OS only, whose functioning is widely tested through the continuous integration workflows. 

You can either install DeepRank2 in a [dockerized container](#dockerfile), which will allow you to test the package and run our [tutorial notebooks](https://github.com/DeepRank/deeprank2/tree/main/tutorials), or you can [install the full version](...) of DeepRank2.
```